roc/address.go

```diff
@@ -0,0 +1,73 @@
+package roc
+
+/*
+#cgo LDFLAGS: -lroc
+#include <roc/receiver.h>
+#include <roc/address.h>
+#include <roc/sender.h>
+#include <roc/log.h>
+#include <stdlib.h>
+*/
+import "C"
+
+import (
+	"runtime"
+	"unsafe"
+)
+
+func stringToCharArray(str string) []C.char {
+	charArray := make([]C.char, len(str))
+	for ind, r := range str {
+		charArray[ind] = (C.char)(r)
+	}
+	return charArray
+}
+
+// NewAddress parses the `ip`, `port` and `family` and initializes the Address object
+func NewAddress(family Family, ip string, port int) (*Address, error) {
+	a := new(Address)
+	a.mem = make([]byte, C.sizeof_roc_address)
+	a.raw = (*C.roc_address)(unsafe.Pointer(&a.mem))
+
+	cfamily := (C.roc_family)(family)
+	ip = safeString(ip)
+	cip := stringToCharArray(ip)
```

Comment:
Let's merge these two functions. Actually we can remove safeString() and just make the char array zero-terminated in stringToCharArray.

And then I'd rename `stringToCharArray` to something like `toCStr` and move to a separate file like `string.go` or `helpers.go`.

Reply:
done